packages/drivers/odsp-driver/src/test/createNewUtilsTests.spec.ts

```diff
@@ -93,7 +93,7 @@ describe("Create New Utils Tests", () => {
 				docId: hashedDocumentId,
 				resolvedUrl,
 			},
-			new TelemetryNullLogger(),
+			createChildLogger(),
```

Comment:
It's a little weird to call `createChildLogger()` with no params.  If you think about it, of course it means the logs sent to it go nowhere, but I think many people may glance at it and not realize this is a no-op logger.

Reply:
it is also weird to have test code in a production package, since this is test, i'd rather the test do weird things, than have the production package do weird things.

Reply:
I feel the same as Mark. Wouldn't necessarily block the change, but aleast we need to document the new functions, and especially this one to clarify what happens when called with no params.

Reply:
i also don't think people are very intentional about which test logger they use, looks more like they picked one, and the rest of the test just keep using it.